templates/all-services/index.json-ld

```diff
@@ -102,6 +104,13 @@
 		}
 	],
 	"iteration": [
+		{
+			"@id": "_:iteration_allservices_3",
+			"name": "All services page - Iteration 3",
+			"date": "2025-07",
+			"additions": [ "Removing expand/collapse all buttons." ],
+			"detectableBy": "Three navigation bands followed by a Services and Information component containing a list of all themes and topics."
+		},
 		{
 			"@id": "_:iteration_allservices_2",
```

Comment:
Add the following at the end of the object:
`"successor": "_:iteration_allservices_3"`